modules/hcp-create-infraenv.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-create-infraenv_{context}"]
+= Create Infraenv
+
+For heterogeneous node pools, you must create the `infraenv` custom resource (CR) for each architecture. For example, for node pools with `x86_64` and `ppc64le` architectures, create the `infraenv` CR for `x86_64` and `ppc64le`.
```

Comment:
If this is required for heterogeneous node pools, should this be a step in the _Creating heterogeneous node pools_ procedure so that people don't miss it?